src/microbe_stage/MicrobeStage.tscn

```diff
@@ -2218,6 +2215,7 @@ __meta__ = {
 }
 
 [node name="PauseMenu" parent="." instance=ExtResource( 11 )]
+visible = false
```

Comment:
This is the opposite of the usual problem, so I guess we can allow this and see how long it is until Godot wants to automatically remove this...